HerPortal/Startup.cs

```diff
@@ -102,7 +104,21 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 // app.UseHsts();
             }
-            
+
+            // Use forwarded headers, so we know which URL to use in our auth redirects
+            // AWS ALB will automatically add `X-Forwarded-For` and `X-Forwarded-Proto`
+            var forwardedHeaderOptions = new ForwardedHeadersOptions
+            {
+                ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto
+            };
+
+            // TODO: We know all traffic to the container is from AWS, but ideally we
+            // would still specify the IP and networks of the ALB here
+            forwardedHeaderOptions.KnownNetworks.Clear();
+            forwardedHeaderOptions.KnownProxies.Clear();
```

Comment:
The risk here is that if someone was able to make requests directly to the app (i.e. not via our trusted load balancer) they would be able to spoof the X-Forwarded headers.